### PR TITLE
Update contents

### PR DESCRIPTION
**Pull request by talmurshidi**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [x] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [ ] all links and images work

### CODE REVIEW

File: assets/hyf-workflow-steps-v1.0.svg
Comment (colevandersWands) [inline assets/hyf-workflow-steps-v1.0.svg:1]: 😺

File: 0-local-development-without-git/vscode/vscode.md
Comment (colevandersWands) [inline 0-local-development-without-git/vscode/vscode.md:1]: change name to README.md?

File: 0-local-development-without-git/npm/npm.md
Comment (colevandersWands) [inline 0-local-development-without-git/npm/npm.md:1]: change name to README.md?

File: 0-local-development-without-git/cli/cli.md
Comment (colevandersWands) [inline 0-local-development-without-git/cli/cli.md:1]: change name to README.md?

File: package.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
   "scripts": {
     "format": "prettier --write ./",
     "format:check": "prettier --check ./",
-    "spell-check": "cspell  './**'",
+    "spell-check": "cspell  \"./**\"",
     "lint:ls": "ls-lint",
     "lint:md": "markdownlint --ignore node_modules --config ./.markdownlint.jsonc --fix ./ "
   },
   "author": "HackYourFuture Belgium",
```
Comment (colevandersWands) [inline package.json]: Change to DeNepo since the license has changed?

**Review by colevandersWands — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (talmurshidi) [reply package.json]: Done.

File: 0-local-development-without-git/cli/cli.md
Comment (talmurshidi) [reply 0-local-development-without-git/cli/cli.md:1]: Done

File: 0-local-development-without-git/npm/npm.md
Comment (talmurshidi) [reply 0-local-development-without-git/npm/npm.md:1]: Done

File: 0-local-development-without-git/vscode/vscode.md
Comment (talmurshidi) [reply 0-local-development-without-git/vscode/vscode.md:1]: Done

**Review by colevandersWands — APPROVED** [review]

**Pull request merged**